src/main/java/gov/acwi/wqp/etl/BasicLookupProcessor.java

```diff
@@ -0,0 +1,17 @@
+package gov.acwi.wqp.etl;
+
+import org.springframework.batch.item.ItemProcessor;
+
+public class BasicLookupProcessor extends BaseProcessor implements ItemProcessor<GwReflist, BasicLookup> {
+	
+	@Override
+	public BasicLookup process(GwReflist gwReflist) throws Exception {
+		final String code = trimString(gwReflist.getGwRefCd());
+		final String name = trimString(gwReflist.getGwRefNm());
+		final String description = trimString(gwReflist.getGwRefDs());
+		final boolean validFlag = gwReflist.getGwValidFg() == 'Y';
+		
+		return new BasicLookup(code, name, gwReflist.getGwSortNu(), description, validFlag);
```

Comment:
Mixed thoughts on this - the "final" seems like noise (but if I knew JAVA, I would know why it is or shouldn't be here...)
- not sure on some of the contructor parms being variables and one a method (I know it isn't wrapped in a function like the others)
- 5 is probably ok, but much more (coupled with the data manipulation and fact that we are in a processor) and I may have not done the constructor for BasicLookup.

Reply:
What would be the alternative? I use this pattern in all of the processors, some of which have a lot more parameters.

Reply:
I think you are correct that "final" is noice and I'm fine with making all of the constructor parms variables. Still not sure what you mean by the second bullet point.

Reply:
Now that I reread the last bullet, are you suggesting that it would be better to use the setters?